room.js

```diff
@@ -1,703 +0,0 @@
-let answer
-let allPossibleDirections = ['north', 'south', 'east', 'west']
-let keypad_answer
-
-
-
-/// MOST IMPORTANT OBJECTS AND FUNCTIONS /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-const readline = require('readline');
-const { userInfo } = require('os');
-const readlineInterface = readline.createInterface({
-    input: process.stdin,
-    output: process.stdout
-})
-
-function ask(questionText) {
-    return new Promise((resolve, reject) => {
-      readlineInterface.question(questionText, resolve);
-    });
-  };
-
-function examine(object){
-    if(object.current_room === player.current_room && object.current_facing === player.current_facing){
-        console.log('\n \x1b[36m%s\x1b[0m', `\n${object.desc}`)
```

Comment:
Things like "'\n \x1b[36m%s\x1b[0m'" are especially important to comment out even though it's just creating styling for the console.log.